### PR TITLE
v1.1.8 Release Notes

### PR DESCRIPTION
**Pull request by KiranSukhavasi**
1.1.8 release notes.

Signed-off-by: Kiran Sukhavasi <kiran.sukhavasi@intel.com>

### CODE REVIEW

**Review by shrikant1407 — APPROVED** [review]

File: release-notes-v1.1.8.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    * Key Exchanges: ECDH256, and ECDH384
+    * Ciphers: AES-CCM-64-128-128, AES-CCM-64-128-256, AES128GCM, and AES256GCM.
+    * Public Key Encoding:  X509.
+    * COSE Signature Types: ES256, and ES384.
+
+  * EPID Verification Service: [epid-verification-service](https://github.com/fido-device-onboard/epid-verification-service) is a wrapper service written on top of the EPID SDK to assist the FDO Rendezvous service and FDO Owner service to perform device signature verification for EPID based devices.
+  
+### New Features
+
+**client-sdk-fidoiot**: Securing the device credentials (secrets) inside TPM as per the spec.
```
Comment (v-vasavi) [inline release-notes-v1.1.8.md]: Let us move this to changed features and reword it to "Storage of credentials in TPM has been modified to be compliant with the TPM spec" then add the disclaimer .. also need to capture FIPS compliance in changed features and add the text from the readme disclaimer.

File: release-notes-v1.1.8.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+### Changes to existing features
+
+**pri-fidoiot**: Added REST API support to update the replacement RV information.
+
+### Fixed Issues
+ 
+**client-sdk-fidoiot**: Cross attestation between CSDK client and PRI services are fixed.
+
+**client-sdk-fidoiot**, **pri-fidoiot** **epid-verification-service**: Updated the required third party dependencies to be complaint with FIPS.
```
Comment (v-vasavi) [inline release-notes-v1.1.8.md]: Let FIPS be in changed features and we need to add the disclaimer for it .. Let us add "The version of third-party dependencies have been updated." here

File: release-notes-v1.1.8.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+### Changes to existing features
+
+**client-sdk-fidoiot**: Storage of credentials in TPM has been modified to be compliant with the TPM spec. Please note that the [FIDO Alliance specification "Securing FDO Credentials in the TPM"](https://fidoalliance.org/specs/FDO/securing-fdo-in-tpm-v1.0-rd-20231010/securing-fdo-in-tpm-v1.0-rd-20231010.html) has been published as a Review Draft by the FIDO Alliance, and is still subject to comment and change. With respect to [section 4.2, Handles for FDO Credentials](https://fidoalliance.org/specs/FDO/securing-fdo-in-tpm-v1.0-rd-20231010/securing-fdo-in-tpm-v1.0-rd-20231010.html#Handles_LABEL), Trusted Computing Group (TCG) has allocated the NVRAM addresses referenced, and is moving towards approval of the persistent object handles.
+
+**pri-fidoiot**: Added REST API support to update the replacement RV information.
+
+**client-sdk-fidoiot**, **pri-fidoiot** **epid-verification-service**: Updated the required third party dependencies to be complaint with FIPS.
+
+*Use of Bouncy Castle FIPS as Security Provider in FDO Project
```
Comment (v-vasavi) [inline release-notes-v1.1.8.md]:
```suggestion


```

File: release-notes-v1.1.8.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+ The PRI FIDO IOT component uses Bouncy Castle FIPS as the primary security provider for all cryptographic operations within the project with the exception of the KDF.
+ The KDF implementation is compliant with the FIDO specification and is not based on the Bouncy Castle FIPS.*
```
Comment (v-vasavi) [inline release-notes-v1.1.8.md]:
```suggestion
 The KDF implementation is compliant with the FIDO specification and is not based on the Bouncy Castle FIPS.

```
```suggestion
 The KDF implementation is compliant with the FIDO specification and is not based on the Bouncy Castle FIPS.

```

**Review by KiranSukhavasi — COMMENTED** [review]
updated.

**Review by DukeDavis12 — DISMISSED** [review]

**Review by DukeDavis12 — DISMISSED** [review]

**Review by DukeDavis12 — APPROVED** [review]

**Review by v-vasavi — APPROVED** [review]
